server/channels/app/login_test.go

```diff
@@ -48,22 +48,33 @@ func TestCWSLogin(t *testing.T) {
 
 	t.Run("Should authenticate user when CWS login is enabled and tokens are equal", func(t *testing.T) {
 		token := model.NewToken(TokenTypeCWSAccess, "")
-		defer th.App.DeleteToken(token)
+
+		defer func() {
+			err := th.App.DeleteToken(token)
+			require.NoError(t, err)
```

Comment:
There are some linter error with your changes:
```
Error: channels/app/login_test.go:54:4: calling require.NoError on *github.com/mattermost/mattermost/server/public/model.AppError, please use require.Nil instead
Error: channels/app/login_test.go:65:3: calling require.NoError on *github.com/mattermost/mattermost/server/public/model.AppError, please use require.Nil instead
Error: channels/app/login_test.go:75:4: calling require.NoError on *github.com/mattermost/mattermost/server/public/model.AppError, please use require.Nil instead
```


You need to use `require.Nil` as `err` is an `*model.AppError` and not an `error`. You can also change the variable name to `appErr` to make that clear.

Please let me know if you have any questions.